packages/workflow/src/records/index.ts

```diff
@@ -35,3 +35,27 @@ export async function getRecordById<T extends Array<keyof StateIdenfitiers>>(
 
   return res.json()
 }
+
+export async function getRecordsOfAllStatus(
```

Comment:
let's rename this to `getValidRecordById` and have the return type of this function to be a `ValidRecord`